python-package/securexgboost/core.py

```diff
@@ -1139,6 +1139,8 @@ def attest(self, verify=True):
             remote_report = response.remote_report
             remote_report_size = response.remote_report_size
 
+            print(pem_key)
```

Comment:
I suppose this is left over from debugging?

Reply:
Fixed